homework/src/style.css

```diff
@@ -1,3 +1,188 @@
+    background-color: gray;
+    color: white;
+    border-radius: 4px;
+}
+
+.h2 {
+    text-align: center; 
+}
+
 .alert-error {
     color: red;
+}
+
+@media (min-width: 100px) and (max-width: 800px) { 
+    body {
+        margin-left: 0;
+        margin-right: 0;
+        margin-top: 30px;
+        width: 100%;
+        font-size: 0.9em;
+        font-family: "RO Sans",Calibri,sans-serif;
+        background-color: #f8f8f8;
+    }
+
+    header {
+        width: 90%;
+        margin-left:5%;
+        margin-right:5%;
+        margin-bottom:10px; 
+        background-color: rgb(0, 153, 255);
+        padding-bottom:10px;
+    }
+
+    section {
+        background-color: rgb(238, 209, 209);
+        display: flex;
+        flex-direction: column;
+        flex-wrap: nowrap;       
+        justify-content: space-around;
+        align-items: flex-start;
+        width:90%;
+        margin-left:5%;
+        margin-right:5%;
+        margin-bottom:10px; 
+    }
+
+    #divInfo {
+        background-color: aliceblue;width: 75%;
+        width:75%;
+        margin: auto;
+        margin-top:20px;
+        margin-left : 13%;
+        margin-right : 13%;
+        margin-bottom:30px;
+        padding:5%;
+        vertical-align: middle;
+    }
+    
+    #divCont {
+        background-color: aliceblue;width: 75%;
+        width:75%;
+        margin: auto;
+        margin-top:20px;
+        margin-bottom:10px;
+        margin-left : 13%;
+        margin-right : 13%;
+        margin-bottom:30px;
+        padding:5%;
+        vertical-align: middle;
+    }
+}
+
+@media (min-width: 801px)  {
+    body {
+        margin-left: 0;
+        margin-right: 0;
+        margin-top: 50px;
+        width: 80%;
+        font-size: 0.9em;
+        font-family: "RO Sans",Calibri,sans-serif;
+        background-color: #f8f8f8;
+    }
+
+    header {
+        width: 90%;
+        margin-left:5%;
+        margin-right:5%;
+        margin-bottom:10px; 
+        background-color: rgb(0, 153, 255);
+        padding-bottom:10px;
+    }
+
+    section {
+        background-color: rgb(169, 192, 212);
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;       
+        justify-content: space-around;
+        align-items: flex-start;
+        width:90%;
+        margin-left:5%;
+        margin-right:5%;
+        margin-bottom:10px; 
+    }
+
+    #divInfo {
+        background-color: aliceblue;
+        width: 75%;
+        width:75%;
+        margin: auto;
+        margin-top:20px;
+        margin-left : 50px;
+        margin-right : 50px;
+        margin-bottom:30px;
+        padding:5%;
+        vertical-align: middle;
+    }
+    
+    #divCont {
+        background-color: aliceblue;
+        width: 75%;
+        width:75%;
+        margin: auto;
+        margin-top:20px;
+        margin-bottom:30px;
+        margin-left : 50px;
+        margin-right : 50px;
+        padding:5%;
+        vertical-align: middle;
+    }
 }
```

Comment:
While your application is responsive, the left-hand table wraps in an undesired way as shown here for an iPhone6. Also, on a mobile device you should trim down a bit of the margins at the edges, given the limited screen space available.

![iphone6](https://user-images.githubusercontent.com/2788771/44981945-6a57b480-af74-11e8-8840-4b182b7292c6.png)

Reply:
Corrected